backend/dal/Repositories/ActivityRepository.cs

```diff
@@ -135,6 +144,16 @@ public bool Delete(long activityId)
                 this.Context.PimsActivityInstanceNotes.Remove(activityNote);
             }
 
+            foreach (var propAcqFile in instance.PimsActInstPropAcqFiles)
+            {
+                this.Context.PimsActInstPropAcqFiles.Remove(propAcqFile);
+            }
+
+            foreach (var propRsrchFile in instance.PimsActInstPropRsrchFiles)
```

Comment:
Not necessarily applicable to the links you changed, but would document, leave orphaned documents if that's the only one they are linked too? maybe the same with notes

Reply:
Need to think about this one for a bit.

Reply:
I think I'm ok with this. the service enforces the business logic to ensure that any related entities (documents, notes) are deleted/disabled when an activity is deleted, whereas this repository is just responsible for making sure that an activity and all of its fk entities are removed properly. The repository shouldn't need to worry about other entities (documents, notes) other then relationships that cause the delete to fail.